crates/uv/tests/pip_install.rs

```diff
@@ -3852,6 +3852,190 @@ fn install_package_basic_auth_from_url() {
     context.assert_command("import anyio").success();
 }
 
+/// Install a package from an index that requires authentication, provided via `UV_BASIC_AUTH_URLS`
+#[test]
+fn install_package_basic_auth_from_UV_BASIC_AUTH_URLS() {
```

Comment:
```suggestion
fn install_package_basic_auth_from_uv_basic_auth_urls() {
```
grep and replace gone a bit too wild I guess 😄

Reply:
Haha thanks